pkg/destroy/aws/aws.go

```diff
@@ -73,32 +78,34 @@ func (o *ClusterUninstaller) Run() error {
 
 	awsConfig := &aws.Config{Region: aws.String(o.Region)}
```

Comment:
This can be moved inside the `if o.Session == nil` block as it is only used there.

Reply:
> This can be moved inside the if o.Session == nil block as it is only used there.

Obsolete with 72319c7f4 -> bb4396613.